lean4-infoview/src/infoview/info.tsx

```diff
@@ -271,8 +271,13 @@ function InfoAux(props: InfoProps) {
 
         function onError(err: any) {
             const errS = typeof err === 'string' ? err : JSON.stringify(err);
-            setError(`Error fetching goals: ${errS}`);
-            setStatus('error');
+            // we need to check if this value is empty or not, because maybe we are assigning
+            // a message error with an empty error
+            if (errS === '{}' || errS === undefined) { setError(undefined); }
```

Comment:
tiny nit, I think this if test formatting would look better if the setError was on a newline.